package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@streammedev/hermes",
+  "version": "0.0.1",
+  "description": "Messages like you have never seen them before",
```

Comment:
Hermes: delivering a composable component that seamlessly adds autocomplete functionality to messaging wherever you place it.